src/components/ShowMoreScholarships.jsx

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react';
+import ScholarshipsContext from '../models/ScholarshipsContext';
+import ShowMoreScholarshipCard from './ShowMoreScholarshipCard';
+import Grid from '@mui/material/Grid';
+
+export default function ShowMoreScholarships({ currentId }) {
+  const info = useContext(ScholarshipsContext);
+  const scholarships = info.scholarships;
+
+  return (
+    <Grid container spacing={2} justify="center">
+      {scholarships
+        .filter((s) => s.id !== currentId)
+        .slice(0, 3)
+        .map((scholarship, id) => {
```

Comment:
You really need to learn to read docs man! The second parameter is **not** the scholarship ID! It's the index position of the element in the list!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#parameters